controllers/edgedeployment_controller_test.go

```diff
@@ -630,5 +631,24 @@ var _ = Describe("Controllers", func() {
 				Expect(res).To(Equal(reconcile.Result{Requeue: true, RequeueAfter: 0}))
 			})
 		})
+
+		Context("Concurrency", func() {
+			It("SplitEdgeDevices", func() {
+				const (
+					nEdgeDevices = 100
```

Comment:
More input cases needed.
I would cover:
- less devices than concurrency;
- exactly the same number of devices as concurrency;
- exactly multiple of the concurrency number of devices (nConcurrency=7; nEdgeDevices=14);
- negative concurrency;
- concurrency zero;
- concurrency one;
- number of devices equals multiple of concurrency-1 (nConcurrency=7; nEdgeDevices=13);

Reply:
I made it exportable only temporarily. It should be **s**plitEdgeDevices.
I apologize for the confusion.